tests/e2e/local/minikube/install_prereqs_debian.sh

```diff
@@ -59,12 +59,12 @@ fi
 
 # Install minikube.
 function install_minikube() {
-  if ! curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.27.0/minikube-linux-amd64 \
-      && chmod +x minikube \
```

Comment:
This && would still be needed

Reply:
Oh, nice finding. 
It's fixed now. PTAL.